src/components/manage/Controlpanels/Controlpanel.jsx

```diff
@@ -141,6 +141,35 @@ class Controlpanel extends Component {
   }
   form = React.createRef();
 
+  // Goal would be to replace the Form schema prop in line 182, with this function
+  filterSchema = (controlpanel) => {
+    const panelType = controlpanel['@id'].replace('/@controlpanels/', '');
+    let unwantedSettings = [];
+
+    // Allows to specify fields to remove per controlpanel
+    switch (panelType) {
+      case 'language':
+        unwantedSettings = ['display_flags', 'always_show_selector'];
+        break;
+      default:
+        unwantedSettings = ['none!'];
+    }
+
+    // Creates modified version of properties
+    const newPropertiesObj = Object.keys(controlpanel.schema.properties)
+      .filter((key) => !unwantedSettings.includes(key))
+      .reduce((obj, key) => {
+        return Object.assign(obj, {
+          [key]: controlpanel.schema.properties[key],
+        });
+      }, {});
```

Comment:
@danalvrz I usually try to avoid using reduce, because I always have to stop and think for a minute to remember how it works. Here is another way to do it that I think is easier to read:
```
const newPropertiesObj = Object.fromEntries(
   Object.entries(controlpanel.schema.properties).filter(
      ([key, val]) => !unwantedSettings.includes(key)
   )
);```